tests/e2e/watch_test.go

```diff
@@ -57,16 +58,18 @@ var _ = Describe("odoWatchE2e", func() {
 						fileModificationPath,
 					)
 					fmt.Println("Received signal, starting file modification simulation")
-					runCmdShouldPass("mkdir -p " + tmpDir + "/nodejs-ex" + "/'.a b'")
-					runCmdShouldPass("mkdir -p " + tmpDir + "/nodejs-ex" + "/'a b'")
-					runCmdShouldPass("touch " + tmpDir + "/nodejs-ex" + "/'a b.txt'")
-
-					runCmdShouldPass(fmt.Sprintf("mkdir -p %s/nodejs-ex/tests/sample-tests", tmpDir))
-					runCmdShouldPass(fmt.Sprintf("touch %s/nodejs-ex/tests/sample-tests/test_1.js", tmpDir))
+					fmt.Println("Received signal, starting file modification simulation")
+					Expect(createDir(tmpDir + "/nodejs-ex" + "/.a")).To(BeNil())
```

Comment:
instead of concatenating strings with `/` you should use `filepath.Join`, this will handle path properly on different platforms like windows

Reply:
true